qchem/pennylane_qchem/qchem/hf/hamiltonian.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions needed for computing the molecular Hamiltonian.
+"""
+# pylint: disable= too-many-branches, too-many-arguments, too-many-locals, too-many-nested-blocks
+import autograd.numpy as anp
+import pennylane as qml
```

Comment:
more pylint issues here (you can run pylint locally on this directory to show them all at once)